test/sql/test_http_get.sql

```diff
@@ -22,3 +22,11 @@ rollback;
     -- from
         -- request;
 -- rollback;
+
+-- TODO run these tests:
```

Comment:
We'll need a `select net.collect_response` under each of these to be confident that the error messages that get output are what we expect

Reply:
> I think error_msg wants to be on net.http_response rather than the request queue.
> In other words, there is no public API to expose error messages in the request queue

Yeah, you're right. I'll change the error message to be on the response table.

Reply:
Hm. You mean I need to change something here? I've just left these as comments for now.

Reply:
They're commented out so no need to change them for now, only mentioning that we'll need to adjust these tests once there's a way to solve the multi-transaction problem